source/both.js

```diff
@@ -12,6 +12,8 @@ import lift from './lift';
  * meaning that the second function will not be invoked if the first returns a
  * false-y value.
  *
+ * Also the method returns not only a function, but any applicative functor of corresponded type.
```

Comment:
I don't think we need this, there is already a note about these functions working with any applicative